sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcIO.java

```diff
@@ -57,14 +59,25 @@
  *
  * pipeline.apply(JdbcIO.read()
  *   .withDataSource(myDataSource)
+ *   .withRowMapper(new JdbcIO.RowMapper<MyElement>() {
+ *     public MyElement mapRow(ResultSet resultSet) {
+ *       // use the resultSet to build the element of the PCollection
+ *       // for instance:
+ *       // return resultSet.getString(2);
+ *     }
+ *   })
  *
  *   }
  * </pre>
+ * <p>
+ *   You can find an full {@link RowMapper} example in {@code JdbcIOTest}.
+ * </p>
  * <h3>Writing to JDBC datasource</h3>
  * <p>
  * JDBC sink supports writing records into a database. It expects a
  * {@code PCollection<T>}, converts the {@code T} elements as SQL statement
- * and insert into the database. T is the type expected by the provided {@link ElementInserter}.
+ * and setParameters into the database. T is the type expected by the
```

Comment:
The grammar seems off - "converts the T elements as SQL statement and setParameters".
Perhaps:
It writes a PCollection<T> to the database by converting each T into a PreparedStatement via a user-provided PreparedStatementSetter?